addons/cargo/functions/fnc_unloadItem.sqf

```diff
@@ -41,9 +46,18 @@ if (_itemSize < 0) exitWith {
     false // return
 };
 
-// This covers testing vehicle stability and finding a safe position
-private _emptyPosAGL = [_vehicle, _item, _unloader] call EFUNC(common,findUnloadPosition);
-TRACE_1("findUnloadPosition",_emptyPosAGL);
+private _deployed = _deploy isNotEqualTo [];
+
+if (!_deployed) then {
+    // This covers testing vehicle stability and finding a safe position
+    for "_i" from 1 to 3 do {
+        _emptyPosAGL = [_vehicle, _item, _unloader] call EFUNC(common,findUnloadPosition);
+
+        if (_emptyPosAGL isNotEqualTo []) exitWith {};
+    };
```

Comment:
I'm not sure why this should be executed more than once?

Reply:
Because `EFUNC(common,findUnloadPosition)` is not guaranteed to return a valid position, despite there one existing.

I'm not a fan of it, but I'm not sure how else to go about it - unless we modify `EFUNC(common,findUnloadPosition)` to always find an existing spot if one exists.

Reply:
> Because EFUNC(common,findUnloadPosition) is not guaranteed to return a valid position, despite there one existing.

Thanks, I hate it.